inc/class-rest-api.php

```diff
@@ -60,6 +60,120 @@ public function register_routes() {
 		);
 	}
 
+	/**
+	 * Add filters for modifying core data or functionality, but only during SST
+	 * REST requests.
+	 */
+	public function add_sst_request_filters() {
+		foreach ( get_post_types() as $post_type ) {
+			add_filter( "rest_pre_insert_{$post_type}", [ $this, 'set_modified' ], 10, 2 );
+		}
+		add_filter( 'wp_insert_post_data', [ $this, 'prevent_updates_from_overwriting_modified_date' ], 5, 2 );
+		add_filter( 'wp_insert_post_data', [ $this, 'set_modified_for_real' ], 10, 2 );
```

Comment:
😆

Reply:
I was really hoping @jakewrfoster would grab the code review for this joke